web/src/components/netflow-traffic.tsx

```diff
@@ -31,55 +22,103 @@ export const NetflowTraffic: React.FC = () => {
   const [isModalOpen, setModalOpen] = React.useState(false);
   const { t } = useTranslation('plugin__network-observability-plugin');
 
+  //TODO: create a number range filter type for Packets & Bytes
+  //TODO: set isSelected values from localstorage saved column ids
   const [columns, setColumns] = React.useState<Column[]>([
-    { id: ColumnsId.date, name: t('Date & time'), isSelected: true, defaultOrder: 1 },
-    { id: ColumnsId.srcpod, name: t('Src pod'), isSelected: true, defaultOrder: 2 },
-    { id: ColumnsId.dstpod, name: t('Dst pod'), isSelected: true, defaultOrder: 3 },
-    { id: ColumnsId.srcnamespace, name: t('Src namespace'), isSelected: true, defaultOrder: 4 },
-    { id: ColumnsId.dstnamespace, name: t('Dst namespace'), isSelected: true, defaultOrder: 5 },
-    { id: ColumnsId.srcport, name: t('Src port'), isSelected: true, defaultOrder: 6 },
-    { id: ColumnsId.dstport, name: t('Dst port'), isSelected: true, defaultOrder: 7 },
-    { id: ColumnsId.protocol, name: t('Protocol'), isSelected: true, defaultOrder: 8 },
-    { id: ColumnsId.bytes, name: t('Bytes'), isSelected: true, defaultOrder: 9 },
-    { id: ColumnsId.packets, name: t('Packets'), isSelected: true, defaultOrder: 10 }
+    {
+      id: ColumnsId.timestamp,
+      name: t('Date & time'),
+      isSelected: true,
+      defaultOrder: 1,
+      filterType: FilterType.DATETIME
+    },
+    {
+      id: ColumnsId.srcpod,
+      name: t('Src pod'),
+      isSelected: true,
+      defaultOrder: 2,
+      filterType: FilterType.POD
+    },
+    {
+      id: ColumnsId.dstpod,
+      name: t('Dst pod'),
+      isSelected: true,
+      defaultOrder: 3,
+      filterType: FilterType.POD
+    },
+    {
+      id: ColumnsId.srcnamespace,
+      name: t('Src namespace'),
+      isSelected: true,
+      defaultOrder: 4,
+      filterType: FilterType.NAMESPACE
+    },
+    {
+      id: ColumnsId.dstnamespace,
+      name: t('Dst namespace'),
+      isSelected: true,
+      defaultOrder: 5,
+      filterType: FilterType.NAMESPACE
+    },
+    {
+      id: ColumnsId.srcaddr,
+      name: t('Src address'),
+      isSelected: true,
+      defaultOrder: 6,
+      filterType: FilterType.ADDRESS
+    },
+    {
+      id: ColumnsId.dstaddr,
+      name: t('Dst address'),
+      isSelected: true,
+      defaultOrder: 7,
+      filterType: FilterType.ADDRESS
+    },
+    {
+      id: ColumnsId.srcport,
+      name: t('Src port'),
+      isSelected: true,
+      defaultOrder: 8,
+      filterType: FilterType.NUMBER
+    },
+    {
+      id: ColumnsId.dstport,
+      name: t('Dst port'),
+      isSelected: true,
+      defaultOrder: 9,
+      filterType: FilterType.NUMBER
+    },
```

Comment:
srcport and dstport are no longer numbers since it can display the service name.